apps/challenge-registry/src/_app-theme.scss

```
@@ -1,13 +1,12 @@
 @use '@angular/material' as mat;
-@use 'palette' as pal;
+@use 'libs/challenge-registry/themes/src/palettes' as pals;
```

```suggestion
@use 'libs/challenge-registry/themes/src/palettes' as palettes;
```
Consider using more descriptive/standardized variable names. This suggestion may apply to other variables defined in this PR.